documentation/reference/document-json-put-format.html

```diff
@@ -49,19 +77,27 @@ <h3 id="array-values">Array values</h3>
 <h3 id="map-values">Map values</h3>
 <p>
 Maps are represented as JSON dictionaries. The key has to be a string.
```

Comment:
The key in the json representation or in the Vespa document definition? The example points in the direction of the former.

Reply:
It's the former, as that's the only thing supported by JSON. Will rewrite to explicitly mention this.